src/ds5/ds5-options.cpp

```diff
@@ -290,6 +290,29 @@ namespace librealsense
         }
     }
 
+    auto_exposure_step_option::auto_exposure_step_option(std::shared_ptr<auto_exposure_mechanism> auto_exposure,
+        std::shared_ptr<auto_exposure_state> auto_exposure_state,
+        const option_range& opt_range)
+        : option_base(opt_range),
+        _auto_exposure_state(auto_exposure_state),
+        _auto_exposure(auto_exposure)
+    {}
+
+    void auto_exposure_step_option::set(float value)
+    {
+        if ((!std::isnormal(_opt_range.step)))
```

Comment:
This should also fail on out of boundaries values - there is `is_valid` in option_base class